libs/docs/shared/src/lib/core-helpers/sections-toolbar/sections-toolbar.component.ts

```diff
@@ -71,6 +74,17 @@ export class SectionsToolbarComponent implements OnInit, OnChanges {
                 })
                 .filter(({ content }) => content.length);
         }
+
+        if (!preparedSearchTerm) {
+            return;
+        }
+
+        const totalItemsCount = this.displayedSections.reduce(
+            (prevValue, currentValue) => prevValue + currentValue.content.length,
+            0
+        );
+
+        this._liveAnnouncer.announce(`${totalItemsCount} search results found.`);
```

Comment:
Can we add `@Input` that accepts callback and that function returns string, so the users can provide their own for purposes of supporting i18n. 

Or we can use our i18n package if it's possible in this case.

Reply:
Its a documentation part. Users don't have access to it